### PR TITLE
Updated models with audit tags

### PR DESCRIPTION
**Pull request by kanmaytacker**
Fixed issues expressed in Code review.

Classes Covered
- Buyers
- Country
- States
- City
- Locality
- Apartments

### CODE REVIEW

File: src/main/java/com/FoodyBuddy/Model/Buyer.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+@Table(name = "Buyer")
+public class Buyer{
+	
+	@Id
+	@Column(name = "id", nullable = false)
+	@GeneratedValue
+	private int id;
+	
+	@Column(name = "apartment_id", nullable = false)
+	@ManyToOne
```
Comment (sairamch04) [inline src/main/java/com/FoodyBuddy/Model/Buyer.java:27]: All of us have done the mistake here . near @ManyToOne annottation @column shouldn't be specified

The code should be:
@ManuToOne
@JoinColumn(name = "apartment_id")
 private Apartment apartment;

@SainathB @SainathB @junzy  Look at this

File: src/main/java/com/FoodyBuddy/Model/Buyer.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+	private String mobileNumber;
+	
+	@Column(name = "email")
+	private String email;
+	
+	@Column(name = "flat_no")
+	private String flatNumber;
+	
+	@Temporal(TemporalType.TIMESTAMP)
+	@Column(name = "created_at")
```
Comment (sairamch04) [inline src/main/java/com/FoodyBuddy/Model/Buyer.java]: While this specifies the the database to use Timestamp datatype, we have to also specify default as current_timespamp @junzy @sanjanagyl @SainathB

File: src/main/java/com/FoodyBuddy/Model/Buyer.java [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+	private Date createdAt;
+	
+	@Temporal(TemporalType.TIMESTAMP)
```
Comment (sairamch04) [inline src/main/java/com/FoodyBuddy/Model/Buyer.java:50]: we have to  also set default  current_timestamp  on update also

File: src/main/java/com/FoodyBuddy/Model/Apartment.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+	@Column(name = "id", nullable = false)
+	@GeneratedValue
+	private int id;
+	
+	@Column(name = "name")
+	private String name;
+	
+	@Column(name = "locality_id", nullable = false)
+	@ManyToOne
+	@JoinColumn(name = "id")
```
Comment (junzy) [inline src/main/java/com/FoodyBuddy/Model/Apartment.java]: remove column and replace name= "locality_id"

**Comment by sairamch04** [comment]
Looks good !

**Pull request merged**